src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/person/MergeService.kt

```diff
@@ -33,12 +43,39 @@ class MergeService(
 
   private suspend fun processMergingOfRecords(mergeEvent: MergeEvent, sourcePersonCallback: () -> PersonEntity?, targetPersonCallback: () -> PersonEntity?) {
     val (sourcePersonEntity, targetPersonEntity) = collectPeople(sourcePersonCallback, targetPersonCallback)
+
+    val operationId = telemetryClient.context.operation.id
```

Comment:
Seems a shame to autowire this into this class just to do this. Maybe if it were in the EventLoggingService?